build/common.sh

```diff
@@ -28,13 +28,13 @@ export GOOS
 GOOS=$(go env GOOS)
 export GOARCH
 GOARCH=$(go env GOARCH)
-DEFAULT_CSV_VERSION="4.16.0"
+DEFAULT_CSV_VERSION="4.17.0"
 CSV_VERSION="${CSV_VERSION:-${DEFAULT_CSV_VERSION}}"
 SKIP_RANGE="${SKIP_RANGE:-""}"
 REPLACES_CSV_VERSION="${REPLACES_CSV_VERSION:-""}"
 LATEST_ROOK_IMAGE="docker.io/rook/ceph:v1.13.0.399.g9c0d795e2"
 ROOK_IMAGE=${ROOK_IMAGE:-${LATEST_ROOK_IMAGE}}
-DEFAULT_BUNDLE_IMAGE=rook/rook-ceph-operator-bundle:"${VERSION}"
+DEFAULT_BUNDLE_IMAGE=quay.io/ocs-dev/rook-ceph-operator-bundle:"${VERSION}"
```

Comment:
```suggestion
DEFAULT_BUNDLE_IMAGE=quay.io/ocs-dev/rook-ceph-operator-bundle:"${VERSION}"
```

Why this change?

Reply:
The default location for the image is `quay.io` now, compared to `docker.io/rook` what we discussed when the bundle image was created. So, updated it now.

Reply:
Ohh only for bundle image right?

Reply:
Yes